### DISCUSSION TITLE
Symfony CHANGELOG.md policy question

### DISCUSSION THREAD

**Discussion by gnito-org** [symfony/symfony#43023] [Q&A] [answered]
Should there be multiple CHANGELOG.md files within the same component?

For example, the Mailer component has a [CHANGELOG.md](https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Mailer/CHANGELOG.md) file in its `/` folder, and there is a CHANGELOG.md file for each of the bridge transports ([Google](https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Mailer/Bridge/Google/CHANGELOG.md), [Mailchimp](https://github.com/symfony/symfony/tree/5.4/src/Symfony/Component/Mailer/Bridge/Mailchimp), etc.) in their respective root folders.

I think that's confusing because the bridges are not versioned and released on their own. The component is versioned and released as a whole entity.

Therefore, may I suggest that CHANGELOG.md files be maintained at the "package" level only, i.e., the entity that can be installed by Composer.

**Comment by mbabker** [comment, marked as answer]
The bridges are indeed standalone packages, i.e. https://packagist.org/packages/symfony/google-mailer and https://packagist.org/packages/symfony/mailchimp-mailer
